data/korath/korath missions.txt

```diff
@@ -257,3 +257,602 @@ mission "Small Scale Delivery"
+			`	Pleased with this success, Akafavah continues. "Korath," it says, gesturing at the two pilots.`
+			`	"I'm a human," you say, "from Earth."`
+			`	Akafavah says suddenly, "Oi! Humanika!" It's unclear what emotion the Korath is expressing, but it is a strong one.`
+			`	They see your confusion and seem to give up on communicating with words. It's time for a different approach.`
+				goto map
+
+			label attack
+			action
+				set "First Contact: Korath Exiles: pointed gun on Far'en Lai"
+			`	You reach for your gun. They move faster than you would have expected possible. One of them strikes a pose with their arms and shouts, "Nag! Nag! Koreshi far'en lai!" Simultaneously, the other dives at you, grabbing the gun out of your hands before you have a chance to fire it.`
+			`	Then, instead of attacking you, the one who took your gun begins waving it in your face and talking very quickly, as if scolding you. It points to the sky with a sweeping gesture, then mimics drawing the gun as you just did. Then it points to the ground, puts the gun on the ground, and says, "Far'en lai ko forei lak'far." Then it picks up the gun and repeats both gestures: at the sky, gun drawn. Down, gun on ground. Then it tosses the gun near your feet and waits, expectantly.`
+				goto map
+
+			label map
+			`	There is a long awkward silence, like a cocktail party where you are standing next to someone but have no idea how to start a conversation. Then one of the Korath stoops down and begins drawing in the sand, a spiral with many arms: the galaxy. Near the center of the spiral, it draws the outline of one of their warships. "Korath soru nar," it says.`
+			choice
+				`	(Draw where human territory is.)`
+					goto draw
+				`	(Pretend not to understand what the map is.)`
+					goto wait
+
+			label draw
+			`	You draw the outline of your own ship on the same side of the core as theirs, but farther out, then draw a line around it marking the extents of human space. The Korath responds by drawing something that looks like a spider, beyond the southern edge of human space. It covers one whole galactic arm. Then, near the top of human space it pushes its finger deep into the ground, and again just north of there, and connects the two holes with a line.`
+			choice
+				`	(Add some details I know to the map.)`
+					goto add
+				`	(Wait to see what else they will draw.)`
+					goto wait
+
+			label add
+			# MZ wrote the parts about spiders, long before the Coalition was added to the game, the spiders are Archons. (so saith ravenshining)
+			`	Near the southern end of human space, you draw a dot with a partial ring around it: the Quarg ringworld, under construction. The Korath respond by both beginning to draw more rings, spread out all over the galaxy, some broken, some complete. Then they draw more of the spiders, again spread out all over the galaxy, and then sprinkle sand over them so the lines are barely visible.`
```

Comment:
It is hard to get an unambiguous mental picture of some of the drawings in this mission. You should consider including scene images.

Reply:
It may be for the best that we don't have them. Some of the pictures, if shown with precision, would give away lore that doesn't exist yet. For example, the locations of Archons and Quarg Ringworlds beyond mapped space. Another problem is the pictures with a Korath's finger pressing into the ground. For those, we would have to decide what Korath fingers look like. Generally, we don't design aliens to that detail, leaving it to the imagination of the reader.

Including pictures *would* be possible, but we'd have to tweak the text somewhat.

Reply:
I have been meaning to make some scene images for some time.  There's a beach near me that would be perfect... My phone camera is getting scratchy though and may no longer be up to the task.

The entire galaky does not need to be in-shot, and it can be taken without a finger -- just the impression left by one.